src/malloc_wrapper.cc

```diff
@@ -35,7 +35,8 @@ void gnu_backtrace()
 	void *buffer[BT_BUFSZ];
 	char **strings;
 
```

Comment:
1st function call argument is an uninitialized value [clang-analyzer-core.CallAndMessage]